### PR TITLE
Add phpstan/phpstan recipe

### PR DESCRIPTION
**Pull request by gremo**
| Q             | A
| ------------- | ---
| License       | MIT
| Packagist     | https://packagist.org/packages/phpstan/phpstan

### CODE REVIEW

File: phpstan/phpstan/1.0/phpstan.neon [hunk 1]
```diff
@@ -0,0 +1,8 @@
+parameters:
+    level: 6
+    paths:
+        - bin/
+        - config/
+        - public/
```
Comment (weaverryan) [inline phpstan/phpstan/1.0/phpstan.neon:6]: I'm not sure about `public/` and `bin/`, though I suppose they don't hurt.

File: phpstan/phpstan/1.0/phpstan.neon [hunk 2]
```diff
@@ -0,0 +1,8 @@
+parameters:
+    level: 6
```
Comment (weaverryan) [inline phpstan/phpstan/1.0/phpstan.neon:2]: Why 6?

**Comment by gremo** [comment]
> Having a recipe for this may make sense, but usually the PR would come from a maintainer of the project, or at least approved by one.

I see, but I dont’t think that the phpstan team have the interest or time to look at this. I can open an issue in their repository linking this PR if you want. The receipe is pretty trivial by the way!

File: phpstan/phpstan/1.0/phpstan.neon (hunk 1 above)
Comment (gremo) [reply phpstan/phpstan/1.0/phpstan.neon:6]: Same as https://github.com/symfony/recipes-contrib/blob/main/squizlabs/php_codesniffer/3.6/phpcs.xml.dist#L13

File: phpstan/phpstan/1.0/phpstan.neon (hunk 2 above)
Comment (gremo) [reply phpstan/phpstan/1.0/phpstan.neon:2]: Default 0 means almost no checks. Max is 10 and it’s even too strictly. The level 6 is on their website in the getting started guide, so a new developer would probably start with that level.

**Review by fabpot — APPROVED** [review]

**Comment by norkunas** [comment]
Why not `phpstan.neon.dist` file was chosen for config? Because now it overrode my project's local config instead of dist config

**Pull request merged**